analysers/trials/dumpSize.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import json
+#import urllib.request
+import urllib2
+import io
+import gzip
+import uuid
+
+from datetime import timedelta
+
+from dataTransformations import Transformations
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark import SparkConf
+
+# Takes arguments: Spark master, Cassandra host, Minio host, path of the files
+sparkMaster = sys.argv[1]
+cassandraHost = sys.argv[2]
+trialID = sys.argv[3]
+minioHost = sys.argv[4]
+filePath = sys.argv[5]
+experimentID = trialID.split("_")[0]
+cassandraKeyspace = "benchflow"
```

Comment:
`cassandraKeyspace`, `srcTable`, `destTable` should be defined as part of the configuration file and passed as parameters. The same applies also to other scripts.